SourceX/display.cpp

```diff
@@ -104,7 +104,7 @@ bool SpawnWindow(const char *lpWindowName, int nWidth, int nHeight)
 #else
 	SDL_DisplayMode mode;
 	SDL_GetDisplayMode(0, 0, &mode);
-	refreshDelay = 1000000 / mode.refresh_rate;
+	refreshDelay = 1000000 / (mode.refresh_rate ?: 60);

```

Comment:
What magic is this `?:`?

Reply:
indeed lol xD

Reply:
but it works as intended, seems to be shortened `(mode.refresh_rate ? mode.refresh_rate : 60)`
I'm impressed!

Reply:
It's common in the languages I'm used to, but looks like MSVC will have none of it :(
I might refactor this a bit to have things a bit more DRY.